doc/buildpack.md

```diff
@@ -1,9 +1,75 @@
 # Oryx Buildpack
 
-Oryx provides a buildpack that runs it, so that Oryx can also be used via the [pack][] tool.
+Oryx provides a buildpack that runs it, so that Oryx can also be used via the Cloud Native `[pack](https://github.com/buildpack/pack)` tool, with your own Azure Container Registry (ACR) using ACR Tasks with the Azure CLI `az acr pack` command.  
 
-## Related images
+# Usage
 
-WIP
+```bash
+### Setup ACR
 
-[pack]: https://github.com/buildpack/pack
+$ az group create -n MyGroup
+$ az acr create -n MyRegistry -g MyGroup --sku basic
+
+### run buildpack with source pulled from current directory
+### deploys container with ready to run web app
+
+$ az acr pack -r MyRegistry -t sample-app .
+
+### Cleanup
+
+$ az group delete -n MyGroup
+```
+
+See the primary Oryx page for an up-to-date [list of languages supported](/README.md). 
+
+# Motovation
+
+At Microsoft, we're excited by the creation of the [Buildpack](http://buildpacks.io) effort to standardize 'source to container' tools and resources.  Because Oryx provides zero-configuration 'source to web application' capability for Azure App Service for Linux, and Azure Container Registry supports OCI images, its a natural fit to offer Oryx as a standalone buildpack.
+
+# Related images
+
+Information, including docker files for both build and run images, can be [found here](https://github.com/Microsoft/Oryx/tree/master/images/pack-builder).
+
+# About the Oryx Project
```

Comment:
Why are we adding this information here? I'd keep it as its own page and refer to it, or just add it to README.md

Reply:
are you referring to the 'about the oryx project?' section?